PyRDF/__init__.py

```diff
@@ -1,5 +1,24 @@
-from .TDataFrame import TDataFrame
+from .RDataFrame import RDataFrame
 from .Node import Node
-from .Proxy import Proxy
+from .Proxy import Proxy as proxy, Backend_env, Backend_conf
 from .Operation import Operation
-from .CallableGenerator import CallableGenerator
\ No newline at end of file
+from .CallableGenerator import CallableGenerator
+
+def use(backend, conf = {}):
+
+    available_backends = [
+    "local",
+    "spark"
+    ]
+
+    future_backends = [
+    "dask"
+    ]
+
+    if backend in future_backends:
+        raise NotImplementedError(" This backend environment will be considered in the future !")
+    elif backend in available_backends:
+        Proxy.Backend_env = backend
```

Comment:
This should be set in `Backend` instead.

Reply:
:+1: